.phpcs.xml.dist

```diff
@@ -19,6 +19,7 @@
 	<rule ref="WordPress-Extra">
 		<exclude name="WordPress.Files.FileName"/>
 		<exclude name="WordPress.NamingConventions.ValidVariableName"/>
+		<exclude name="WordPress.NamingConventions.PrefixAllGlobals"/>
```

Comment:
How come we're adding a new rule?

Reply:
We're excluding a rule. It gets activated when the command-line arg is passed, or a prefix is set in the PHPCS config file. In this case, the arg is passed to fix a bug, but we don't need the Prefix-checking behaviour to actually run here.

See https://github.com/WordPress/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#naming-conventions-prefix-everything-in-the-global-namespace